client-v2/src/components/util/RefreshPeriodically.tsx

```diff
@@ -0,0 +1,24 @@
+import React, { useEffect, useState } from 'react';
+
+const refreshPeriodically = ({
+  timeMs,
```

Comment:
minor: would `rate` is more intention revealing here?

Reply:
👍